scripts/completions/install.py

```diff
@@ -189,6 +224,107 @@ def _update_rc(
         assert mode == "uninstall"
 
 
+def _update_conda_scripts(
+    commands: List[str],
+    completions_dir: pathlib.Path,
+    mode: ConfigureMode,
+) -> None:
+    """Try to add a `source /.../completions/setup.{shell}` line automatically to conda's activation scripts.
+
+    Args:
+        completions_dir: Path to location of this script.
+        mode: Install or uninstall completions.
+    """
+
+    # Install or uninstall `source_line`.
+    activate_source_lines = _get_source_script(completions_dir, None)
+    deactivate_source_lines = _get_deactivate_script(commands, None)
+
+    conda_path = pathlib.Path(os.environ["CONDA_PREFIX"])
+    activate_path = conda_path / "etc/conda/activate.d/nerfstudio_activate.sh"
+    deactivate_path = conda_path / "etc/conda/deactivate.d/nerfstudio_deactivate.sh"
+    if mode == "uninstall":
+        if activate_path.exists():
+            os.remove(activate_path)
+        if deactivate_path.exists():
+            os.remove(deactivate_path)
+        CONSOLE.log(f":broom: Existing completions uninstalled from {conda_path}.")
+    elif mode == "install":
+        # Install completions.
+        activate_path.parent.mkdir(exist_ok=True, parents=True)
```

Comment:
On my machine, needed to add this for deactivate_path as well

Reply:
Sorry I didn't realise that. Added now.